### ISSUE TITLE
How to convey that admonitions are side-comments that may be skipped (in most cases)

### ISSUE THREAD

**Issue by lesteve** [INRIA/scikit-learn-mooc#284]
There is this weird contradiction that most of the time:
- we use admonitions are side-comments
- because they are colored brings more attention to them than they should. One example off the top of my head that we have in plenty of places:
![image](https://user-images.githubusercontent.com/1680079/115937601-ab3fc700-a498-11eb-9b77-4588a0dfa582.png)

Ideas of the top of my head:
- technically simpler but less visually appealing: can some admonitions advertising more easily that they are side-comments maybe with a less visible color e.g .greyish?
- technically harder but more visually appealing: is there a way to have a foldable admonition so that you see the title only and you can click on it to see more? Can we support a similar thing inside notebooks through our py-to-ipynb conversion logic in https://github.com/INRIA/scikit-learn-mooc/blob/bc157356571cddc9de6fba978e9446cecee1a0da/build_tools/convert-python-script-to-notebook.py#L120 

Minor comments:
- I am pretty sure sometimes (quite rarely though I would say) we use admonitions as way to highlight that something is important so this
- I guess there are more things than admonitions, like footnotes or sidenotes but supporting in notebooks seems to hard
- does O'Reilly do something particular for this kind of stuff and his

**Comment by GaelVaroquaux** [comment]
Reduce their size (slightly smaller font, not by much).

Also, a more great color would work.

**Comment by lesteve** [comment]
Good point for the JupyterBook! I don't think that would work inside a notebook hence for the MOOC or when we give the course ...

At one point I tried to see if there was a way to use some CSS in notebooks and got nowhere. From what I remember you can have a custom.css in your home somewhere, but this does not seem really practical here. In other words we want a custom.css that is included somehow in the notebook we give to people not an additional step that people have to do to install our custom CSS.

**Comment by lesteve** [comment]
Actually scratch most of my previous comment, since we have some custom logic we could hard-code some font in the generated HTML directives: https://github.com/INRIA/scikit-learn-mooc/blob/bc157356571cddc9de6fba978e9446cecee1a0da/build_tools/convert-python-script-to-notebook.py#L79

**Comment by GaelVaroquaux** [comment]
Or, is it easier to insert the "<small>" HTML tag?

**Comment by lesteve** [comment]
`<small>` would probably be the simplest.

Just a note (i.e. for later) the folded note (aka drop-down note) already exists in JupyterBook:
https://jupyterbook.org/content/content-blocks.html#dropdown-admonitions

It seems like this is based on the `dropdown` CSS class so if we wanted to go down this route, we would probably have to hard-code the CSS our MyST to HTML conversion ...
